src/clr-core/styles/tokens/tokens.yml

```diff
@@ -1180,7 +1180,7 @@ props:
     type: 'hsl'
     category: 'color'
   - name: warning-800
-    value: 'hsl(41, 100%, 36%)'
+    value: 'hsl(41, 100%, 34%)'
```

Comment:
I had to nudge this dark orange down a little so that it could live on top of the warning yellow color we use in a number of places. this gives it 3:1 contrast against that yellow